frontend/src/bundles/cv-editor/components/online-editor/online-editor-handler.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx';
+import React, { useCallback, useState } from 'react';
+
+import { type TabsProps as TabsProperties } from '../../types/interface';
+import css from './online-editor-handler.module.scss';
+
+const OnlineEditorTabsHandler: React.FC<TabsProperties> = ({ tabs }) => {
+    const [activeTab, setActiveTab] = useState(0);
```

Comment:
Judging by variables names, I would expect to have a tab object here. 
Consider using `activeTabIndex` name